src/server/wasm/wasm_registry.h

```diff
@@ -54,16 +54,16 @@ class WasmRegistry {
         : instance_{instance}, store_(store) {
     }
 
-    void operator()() {
+    std::string operator()(std::string_view export_func_name) {
```

Comment:
Now with the SDK we can export multiple functions per module via:

`DF_EXPORT` macro. These are linked automatically so we only need the function name to call it (no other gluing is required)